core/build.gradle

```diff
@@ -9,7 +9,8 @@ idea.module.excludeDirs = [
         file('build/libs')]
 
 dependencies {
-    compile  "org.springframework:spring-core:$springVersion",
```

Comment:
It seems that this commit rearranged the dependencies. Is there any reason for that?